README.rst

```diff
@@ -392,10 +392,76 @@ returned by the server in the specified time. For example:
 
 This will cause both add_event() and add_events() to timeout after 100 seconds. If this timeout limit is hit, a requests.Timeout will be raised. Due to a bug in the requests library, you might also see an SSLError (https://github.com/kennethreitz/requests/issues/1294)
 
-Create Scoped Keys
+Create Access Keys
 ''''''''''''''''''
 
-The Python client enables you to create `Scoped Keys <https://keen.io/docs/security/#scoped-key>`_ easily. For example:
+The Python client enables the creation and manipulation of `Access Keys <https://keen.io/docs/access/access-keys>`_. Examples:
+
+.. code-block:: python
+
+    import keen
+
+    # Master key must be set in an environment variable ahead of time.
+
+    # Create an access key.
```

Comment:
In the comment, could you add a link to permissions section in the docs? It would be handy since there's a lot of different permissions.

Reply:
Maybe this section? https://keen.io/docs/access/access-keys/#customizing-your-access-key

Reply:
Thanks. Will do.